backoffice/backoffice/workflows/api/views.py

```diff
@@ -116,11 +116,15 @@ def create(self, request):
                 data=serializer.validated_data["data"],
                 workflow_type=serializer.validated_data["workflow_type"],
             )
+        workflow["data"][
+            "$schema":"https://inspirehep.net/schemas/records/authors.json"
```

Comment:
let's make it config variable, just in case we would like to change it in the future